.github/workflows/ci.yml

```diff
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [18.x, 20.x, 22.x]
+        node-version: [20.x, 22.4]
```

Comment:
18.x was removed because it doesn't support `process.loadEnvFile()`, `22.4` is temporarily locked due to bug in `22.5.0` (and `22.5.1` is not autodownloaded yet for some reason).